tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -478,6 +478,12 @@ def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
                     get_jax_decoder(args['input_model'], args)
                 else:
                     raise Error("JAX Frontend is not available.")
+            if model_framework == "tf":
+                try:
+                    from nncf.tensorflow.strip import strip
```

Comment:
please check that nncf module is installed. Only if installed, apply this code

Reply:
I've already used the try-except block to check if the `nncf` module is installed. Or do you suggest using another method?

Reply:
No, it is not working for me. Let us handle this properly. We can gracefully check it.